packages/dds/task-manager/src/test/taskManager.spec.ts

```diff
@@ -58,399 +59,528 @@ describe("TaskManager", () => {
-
-            taskManager1.abandon(taskId);
-            assert.ok(!taskManager1.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-            const revolunteerTaskP = taskManager1.volunteerForTask(taskId);
-            assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-            containerRuntimeFactory.processAllMessages();
-            const isAssigned2 = await revolunteerTaskP;
-            assert.ok(isAssigned2, "Should resolve true");
-            assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(taskManager1.assigned(taskId), "Should be assigned");
-        });
-
-        it("Can attempt to volunteer for task twice and abandon twice (after ack)", async () => {
-            const taskId = "taskId";
-            const volunteerTaskP1 = taskManager1.volunteerForTask(taskId);
-            assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-            containerRuntimeFactory.processAllMessages();
-            const isAssigned1 = await volunteerTaskP1;
-            assert.ok(isAssigned1, "Should resolve true");
-            assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(taskManager1.assigned(taskId), "Should be assigned");
-
-            const volunteerTaskP2 = taskManager1.volunteerForTask(taskId);
-            assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(taskManager1.assigned(taskId), "Should be assigned");
-            containerRuntimeFactory.processAllMessages();
-            const isAssigned2 = await volunteerTaskP2;
-            assert.ok(isAssigned2, "Should resolve true");
-            assert.ok(taskManager1.queued(taskId), "Should be queued");
-            assert.ok(taskManager1.assigned(taskId), "Should be assigned");
-
-            taskManager1.abandon(taskId);
-            assert.ok(!taskManager1.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-            containerRuntimeFactory.processAllMessages();
-            assert.ok(!taskManager1.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-
-            taskManager1.abandon(taskId);
-            assert.ok(!taskManager1.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-            containerRuntimeFactory.processAllMessages();
-            assert.ok(!taskManager1.queued(taskId), "Should not be queued");
-            assert.ok(!taskManager1.assigned(taskId), "Should not be assigned");
-        });
+        describe("Volunteering for a task", () => {
```

Comment:
The reorganizing is probably fine, though it seems to make the diff pretty mad :-P

Maybe consider putting up a quick PR with just the reorganization that we can check in (I can manage though if it's too much trouble).